src/style.css

```diff
@@ -1,3 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  --hover-color: #00acdc;
+  margin: 0;
+}
+
+.header__nav {
+  /* width: 100%; */
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* position: relative; */
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo__img {
+  position: absolute;
```

Comment:
You don't need to change the document flow for the logo block. Absolute positioning is redundant here

Reply:
Understood, thank you. I just took it from Figma:)

Reply:
> You don't need to change the document flow for the logo block. Absolute positioning is redundant here

Just wanted to confirm: 

1. without position: absolute, the left and top is not working for me and I cannot pass the test
2. shall I try to achieve the result with padding or margin?

Reply:
<img width="790" alt="Screensh
<img width="786" alt="Screenshot 2023-06-18 at 19 54 33" src="https://github.com/mate-academy/layout_moyo-header/assets/100079561/b2169302-154e-4f35-ad83-ab68156b958a">
ot 2023-06-18 at 19 55 06" src="https://github.com/mate-academy/layout_moyo-header/assets/100079561/bf231b0a-5d1b-4a86-aaad-1ab48a7a2ad0">

Reply:
<img width="765" alt="Screenshot 2023-06-18 at 20 10 57" src="https://github.com/mate-academy/layout_moyo-header/assets/100079561/ac207661-2733-485d-94fe-4b20984e8b80">